lib/ruby_indexer/lib/ruby_indexer/index.rb

```diff
@@ -56,8 +58,8 @@ def register_included_hook(module_name, &hook)
       (@included_hooks[module_name] ||= []) << hook
     end
 
-    sig { params(uri: URI::Generic).void }
-    def delete(uri)
+    sig { params(uri: URI::Generic, skip_require_tree: T::Boolean).void }
+    def delete(uri, skip_require_tree: false)
```

Comment:
nit: I prefer naming it `skip_require_paths_tree` so when searching for `require_paths_tree`, this flag jumps out too.